src/tenantDataMigrations/update-tenant-column-script.js

```diff
@@ -0,0 +1,597 @@
+				`)
+				console.log(`✅ Set organization_code NOT NULL for ${tableName}`)
+			} catch (error) {
+				console.log(`❌ Error setting organization_code NOT NULL for ${tableName}: ${error.message}`)
+			}
+		}
+	}
+
+	/**
+	 * Phase 2: Update primary key constraints
+	 */
+	async updatePrimaryKeys() {
+		console.log('\n🔑 PHASE 2: Updating primary key constraints...')
+		console.log('='.repeat(50))
+
+		const primaryKeyConfigs = {
+			connection_requests: 'tenant_code, user_id, friend_id',
+			connections: 'tenant_code, user_id, friend_id',
+			entities: 'tenant_code, id, entity_type_id',
+			entity_types: 'tenant_code, id, organization_id',
+			forms: 'tenant_code, id, organization_id',
+			organization_extension: 'tenant_code, organization_id',
+			user_extensions: 'tenant_code, user_id',
+			question_sets: 'code, tenant_code',
+			questions: 'id, tenant_code',
+			report_queries: 'report_code, tenant_code',
+			report_types: 'title, tenant_code',
+			// Default for all other tables
+			default: 'tenant_code, id',
+		}
+
+		for (const tableName of this.allTables) {
+			try {
+				// Check if table exists
+				const [tableExists] = await this.sequelize.query(`
+					SELECT EXISTS (SELECT FROM information_schema.tables WHERE table_name = '${tableName}') as exists
+				`)
+
+				if (!tableExists[0].exists) {
+					console.log(`⚠️  Table ${tableName} does not exist, skipping`)
+					continue
+				}
+
+				// Drop existing primary key
+				await this.sequelize.query(`
+					ALTER TABLE ${tableName} DROP CONSTRAINT IF EXISTS ${tableName}_pkey CASCADE
+				`)
+
+				// Get primary key configuration
+				const primaryKeyColumns = primaryKeyConfigs[tableName] || primaryKeyConfigs['default']
+
+				// Add new primary key with tenant_code
+				await this.sequelize.query(`
+					ALTER TABLE ${tableName} ADD PRIMARY KEY (${primaryKeyColumns})
+				`)
+
+				console.log(`✅ Updated primary key for ${tableName}: (${primaryKeyColumns})`)
+				this.stats.primaryKeysUpdated++
+			} catch (error) {
+				console.log(`❌ Error updating primary key for ${tableName}: ${error.message}`)
+			}
+		}
+	}
+
+	/**
+	 * Phase 3: Add foreign key constraints
+	 */
+	async addForeignKeys() {
+		console.log('\n🔗 PHASE 3: Adding foreign key constraints...')
+		console.log('='.repeat(50))
+
+		const foreignKeyConfigs = [
+			{
+				table: 'entities',
+				column: 'entity_type_id',
+				refTable: 'entity_types',
+				refColumn: 'id',
+				name: 'fk_entities_entity_type_id',
+			},
+			{
+				table: 'session_attendees',
+				column: 'session_id',
+				refTable: 'sessions',
+				refColumn: 'id',
+				name: 'fk_session_attendees_session_id',
+			},
+			{
+				table: 'feedbacks',
+				column: 'session_id',
+				refTable: 'sessions',
+				refColumn: 'id',
+				name: 'fk_feedbacks_session_id',
+			},
+			{
+				table: 'resources',
+				column: 'session_id',
+				refTable: 'sessions',
+				refColumn: 'id',
+				name: 'fk_resources_session_id',
+			},
+			{
```

Comment:
@sumanvpacewisdom how many times we need comment, questions and questions set is required